client/components/Categories/index.js

```diff
@@ -13,7 +13,7 @@ export default class Categories extends Component {
   createCategories = () => {
     let id = -1;
     return this.props.categories.map(category =>
-        <li key={(id++)} id={id} className={this.props.chosenCategoryId == id ? 'category chosen-category': 'category'} onClick={this.chooseCategory} >
+        <li style={{backgroundColor:this.props.colorMap[category.categoryId]}} key={(id++)} id={id} className={this.props.chosenCategoryId == id ? 'category chosen-category': 'category'} onClick={this.chooseCategory} >
```

Comment:
What if categoryId is bigger ? We need some modulo to repeat colors after exceeding max numbers of colors.

Reply:
i have added that, must missed it while testing :D